xonsh/tools.py

```diff
@@ -830,14 +830,22 @@ def command_not_found(cmd):
     """Uses the debian/ubuntu command-not-found utility to suggest packages for a
     command that cannot currently be found.
     """
+    # lazy import
+    import shutil
+
     if not ON_LINUX:
         return ""
-    elif not os.path.isfile("/usr/lib/command-not-found"):
+
+    cnf = shutil.which("command-not-found")
+    if not cnf:
```

Comment:
This should probably better be `if cnf is None` since the `which` function can only return a path or `None`.

Reply:
Done.

Reply:
I can't see any changes. I guess you forgot to force-push the commit...

Reply:
Sorry for that.

Reply:
Thank you!

Reply:
eh, dirty commit. Again, sorry for that. Should not commit anything on a Friday afternoon...

Reply:
Yeah, same here :wink: